plugins/field-slider/test/field_slider_test.mocha.js

```diff
@@ -6,11 +6,10 @@
 
 const {assert} = require('chai');
 const sinon = require('sinon');
-const {assertSliderField,
-  assertSliderFieldDefault} = require('../test/field_slider_test_helpers');
+const {assertSliderField,assertSliderFieldDefault} = require(
```

Comment:
nite: space between asserFieldSlider and assertSliderFieldDefault

Reply:
Done